packages/python-peid/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-peid
+_pyname=peid
+pkgver=1.2.9
+pkgrel=1
+pkgdesc='Python implementation of the Packed Executable iDentifier (PEiD)'
+arch=('any')
+grouos=('blackarch' 'blackarch-binary' 'blackarch-reversing')
+license=('GPL3')
+depends=('python' 'python-pefile' 'python-pip')
+makedepends=('python-setuptools')
+source=("$pkgname::git+https://github.com/packing-box/peid")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  cat src/peid/VERSION.txt
```

Comment:
if a git package use one of https://github.com/BlackArch/blackarch-pkgbuilds/blob/8c072641adc8c758c4de8babb9285faee0501cd4/PKGBUILD-generic-git#L17-L25

Reply:
The version of peid uses `src/peid/VERSION.txt` according to https://github.com/packing-box/peid/blob/main/src/peid/__info__.py#L17-L18
I'm really confused which kind of version should be used to meet the version rule of BlackArch.

Reply:
when pypi package source we use a fixed version (eg. semver) but when the source is git, it's mandatory to use one of the two command I linked, for auto-updates and more.

Reply:
Now I've migrated this package to pypi since it is available in pypi, and followed the version from pypi.
I was discovered this project in GitHub and mistakenly made this package as a git package.